### PR TITLE
(refactor) connect profile pic to db on register

### PR DESCRIPTION
**Pull request by gregthompson27**

### CODE REVIEW

File: server/api/cloudinaryRoutes.js [hunk 1]
```diff
@@ -8,13 +8,20 @@ imageRouter.get('/', (req, res) => {
   res.send('You made a get request but you ain\'t getting nothin\' back!');
 });
 
+imageRouter.get('/newProfilePicture', async (req, res) => {
+  const { image } = req.query;
+  const uploadedImage = await cloudinary.uploader
+    .upload(image);
+  console.log('url about to be sent back:', uploadedImage.url);
```
Comment (jstrzesz) [inline server/api/cloudinaryRoutes.js]: remove console.log

File: server/api/cloudinaryRoutes.js [hunk 2]
```diff
@@ -8,13 +8,20 @@ imageRouter.get('/', (req, res) => {
...
+imageRouter.get('/newProfilePicture', async (req, res) => {
+  const { image } = req.query;
+  const uploadedImage = await cloudinary.uploader
+    .upload(image);
+  console.log('url about to be sent back:', uploadedImage.url);
+  res.send(uploadedImage.url);
+});
+
 imageRouter.post('/profile', async (req, res) => {
   console.log('post route got hit');
```
Comment (jstrzesz) [inline server/api/cloudinaryRoutes.js]: remove console.log

File: server/routes/jwtAuth.js [hunk 3]
```diff
@@ -52,12 +55,10 @@ router.post("/login", validEmail, async (req, res) => {
...
     const validPassword = await bcrypt.compare(password, user.password);
-    console.log("password is valid?", validPassword);
     if (!validPassword) {
       return res.status(401).json("Invalid Password, line 58");
     }
     const jwtToken = generateToken(user.id);
-    console.log({ jwtToken });
     res.json({ jwtToken });
   } catch (err) {
     console.error(err.message);
```
Comment (jstrzesz) [inline server/routes/jwtAuth.js]: change to console.warn

File: src/components/landing/auth/Signup.tsx [hunk 4]
```diff
@@ -37,43 +37,39 @@ const SignUp: React.FC<AuthProps> = ({ handleLogin: [isAuthenticated, setAuth] }
...
+      },
+    })
+      .then(({ data }) => {
+        setProfilePhotoUrl(data);
       })
-        .then((results) => console.log(results));
-    } catch (error) {
-      console.error(error);
-    }
+      .catch((error) => console.error(error));
```
Comment (jstrzesz) [inline src/components/landing/auth/Signup.tsx]: change to console.warn

File: src/components/landing/auth/Signup.tsx [hunk 5]
```diff
@@ -37,43 +37,39 @@ const SignUp: React.FC<AuthProps> = ({ handleLogin: [isAuthenticated, setAuth] }
...
+    setSelectedFile(image);
+    setFileInputState(e.target.value);
     const reader = new FileReader();
-    reader.readAsDataURL(selectedFile);
+    reader.readAsDataURL(image);
     reader.onloadend = () => {
       uploadImage(reader.result);
     };
     reader.onerror = () => {
       console.error('reader experienced an error');
```
Comment (jstrzesz) [inline src/components/landing/auth/Signup.tsx]: console.warn

**Review by slatzz — APPROVED** [review]

**Review by jessrichmond — APPROVED** [review]

**Pull request merged**